### DISCUSSION TITLE
[IT] Add "IO" platform

### DISCUSSION THREAD

**Discussion by libremente** [publiccodeyml/publiccode.yml#116] [General]
We should add:
- [ ] `io` 
- [x] `pagopa` 

(or both) to the platforms sections

**Comment by bfabio** [comment]
~~Posted to the forum as https://publiccode.discourse.group/t/add-platform-io-pagopa-issue-87/63~~

Moved to GitHub Discussions

<sub>[Why?](https://github.com/publiccodeyml/publiccode.yml/discussions/104)</sub>

**Comment by bfabio** [comment]
`piattaforme/pagopa` is [already there](https://yml.publiccode.tools/country.html#key-piattafome-pagopa)

**Reply by libremente** [reply]
Yep, we should add `IO` though

**Reply by libremente** [reply]
Added a checklist at the top
